<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255,255,255,255);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="_boutons">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="odometry_button">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #333;border: 2px solid #2ECC71;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #2ECC71 stop: 1 #2ECC71);min-width: 80px;

</string>
          </property>
          <property name="text">
           <string>ODOMETRY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="color_segment_button">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #333 ;border: 2px solid #26C281;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #26C281 stop: 1 #26C281);min-width: 80px;

</string>
          </property>
          <property name="text">
           <string>COLOR AND SEGMENT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stereo_button">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #333;border: 2px solid #4DAF7C;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #4DAF7C stop: 1 #4DAF7C);min-width: 80px;</string>
          </property>
          <property name="text">
           <string>STEREO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="real_button">
          <property name="minimumSize">
           <size>
            <width>84</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #333;border: 2px solid #03A678;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #03A678 stop: 1 #03A678);min-width: 80px;</string>
          </property>
          <property name="text">
           <string>REAL VIEW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quit_button">
          <property name="minimumSize">
           <size>
            <width>94</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Lucida Grande</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #333;border: 2px solid #26A65B;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #26A65B, stop: 1 #26A65B);min-width: 80px; margin-left:10px;</string>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QLabel" name="image1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="image2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QWidget" name="info_odo" native="true">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>230</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(244, 208, 63,150);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="layout_info_odo">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_odo_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Lucida Grande</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>Distance Parcourue</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_odo_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Lucida Grande</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>X : 0.0 m</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_odo_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Lucida Grande</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string>Y : 0.1 m</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="both_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>250</width>
         <height>220</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #333;border: 2px solid #D24D57;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #D24D57 stop: 1 #D24D57 );min-width: 80px;</string>
       </property>
       <property name="text">
        <string>SHOW RESULT:
No</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QSlider" name="slider_alpha">
       <property name="geometry">
        <rect>
         <x>598</x>
         <y>30</y>
         <width>81</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/*background-color: rgba(255, 255, 255, 0);*/s

 QSlider::groove:vertical {
     background: rgb(44,62,80);
     position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
     left: 4px; right: 4px;
 }

 QSlider::handle:vertical {
     height: 10px;
     background: rgb(44,62,80);
     margin: 0 -4px; /* expand outside the groove */
 }

 QSlider::add-page:vertical {
     background: rgb(65,131,215);
 }

 QSlider::sub-page:vertical {
     background: rgb(228,241,254);
 }s

</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>125</number>
       </property>
       <property name="sliderPosition">
        <number>125</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="log_box" native="true">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>200</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(107, 185, 240,220);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="log_box_layout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="log_title">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Lucida Grande</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>Log Box</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="log_content">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Lucida Grande</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string>ERROR </string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="error_continue">
              <property name="styleSheet">
               <string notr="true">color: #333 ;border: 2px solid #C8F7C5;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #C8F7C5 stop: 1 #C8F7C5);min-width: 80px;

</string>
              </property>
              <property name="text">
               <string>Continue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="error_run_thread">
              <property name="styleSheet">
               <string notr="true">color: #333 ;border: 2px solid #FDE3A7;background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35, stop: 0 #FDE3A7 stop: 1 #FDE3A7);min-width: 80px;

</string>
              </property>
              <property name="text">
               <string>Start the thread</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <zorder>image1</zorder>
      <zorder>image2</zorder>
      <zorder>slider_alpha</zorder>
      <zorder>both_button</zorder>
      <zorder>info_odo</zorder>
      <zorder>log_box</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
